atomic_reactor/schemas/config.json

```diff
@@ -451,6 +468,10 @@
                 "description": "Registry url",
                 "type": "string"
             },
+            "organization": {
```

Comment:
I wonder if we want a "definition" for a registry, so we can reference it from both of these places.

Reply:
Probably a good idea.

Reply:
ack will do